source/develop/developer-guide/vdsm/developers.html.md

```diff
@@ -36,18 +36,22 @@ VDSM requires Python 2 as your /usr/bin/python.
 
 Install the following packages before you attempt to build VDSM:
 
-       yum install make autoconf automake pyflakes logrotate python-pep8 libvirt-python python-devel \
+       PYTHON3=python3
+       YUM=dnf
+       $YUM install make autoconf automake pyflakes logrotate python-pep8 libvirt-python python-devel \
```

Comment:
why maintain here a list of dependencies while they're listed in automation/*.packages?
something like "yum install $(cat automation/build*.packages)" will avoid the need to keep this list aligned manually.